targets/XPlatCppSdk/templates/PlayFab_Api.cpp.ejs

```diff
@@ -70,6 +92,24 @@ namespace PlayFab
             AttributeInstall(request, nullptr, nullptr);
         }
     }
-<% } %>}
+<% } %>
+    bool PlayFab<%- api.name %>API::IsSuccessfulResult(PlayFabResultCommon& resultCommon, CallRequestContainer& container)
+    {
+        if (container.errorWrapper.HttpCode == 200)
+        {
+            resultCommon.FromJson(container.errorWrapper.Data);
+            resultCommon.Request = container.errorWrapper.Request;
+            return true;
+        }
+        else // Process the error case
+        {
+            if (PlayFabSettings::globalErrorHandler != nullptr)
```

Comment:
I applaud moving lines into a shared utility function when possible, and reducing the template-induced copy-paste.
In this case, I'm a bit concerned about a function named "IsSuccessfulResult", which executes callbacks when it will return false.
The work done by the function doesn't match it's name.